src/dpi.rs

```diff
@@ -228,6 +228,23 @@ impl<P: Pixel, X: Pixel> Into<[X; 2]> for LogicalPosition<P> {
     }
 }
 
+#[cfg(feature = "mint")]
+impl<P: Pixel> From<mint::Point2<P>> for LogicalPosition<P> {
+    fn from(mint: mint::Point2<P>) -> Self {
+        Self::new(mint.x, mint.y)
+    }
+}
+
+#[cfg(feature = "mint")]
+impl<P: Pixel> Into<mint::Point2<P>> for LogicalPosition<P> {
```

Comment:
Could this be changed to `impl From<LogicalPosition> for mint`? So we have both `From`/`Into` for this direction as well.
Applies for the other occasions as well.